app/feature/feature-insurances/src/main/kotlin/com/hedvig/android/feature/insurances/insurancedetail/ContractDetailDestination.kt

```diff
@@ -153,8 +152,7 @@ private fun ContractDetailScreen(
             contentPadding = WindowInsets
               .safeDrawing
               .only(WindowInsetsSides.Horizontal + WindowInsetsSides.Bottom)
-              .asPaddingValues()
-              .plus(PaddingValues(top = 16.dp)),
+              .asPaddingValues(),
```

Comment:
Will this touch the top now without this extra padding? Could it be that it needs to be 8.dp instead? We do that in some places I think, where it's 8